test/test_common/test_time.cc

```diff
@@ -10,6 +10,12 @@ DangerousDeprecatedTestTime::DangerousDeprecatedTestTime() {}
 
 namespace Event {
 
+TestTimeSystem& GlobalTimeSystem::timeSystem() {
+  // TODO(#4160): Switch default to SimulatedTimeSystem.
+  auto make_real_time_system = []() -> TestTimeSystem* { return new TestRealTimeSystem; };
```

Comment:
Total nit, but since it looks like you might need to fix CI, I would have this return a unique_ptr